lighthouse-core/lib/i18n/locales/zh.json

```diff
@@ -383,6 +383,15 @@
   "lighthouse-core/audits/apple-touch-icon.js | title": {
     "message": "提供了有效的 `apple-touch-icon`"
   },
+  "lighthouse-core/audits/autocomplete.js | description": {
```

Comment:
this seems unrelated and perhaps out of date. was this updated with old scripts?

Reply:
ah was this a result of "This also includes an i18n import."?

Reply:
Not simply. For the new keys (but old strings) to be in all the locales, an i18n roundtrip was necessary.

Reply:
It's not out of date, it's just the string is very new and hasn't been translated yet.

Reply:
I gotcha, you did an i18n roundtrip to preserve the strings that got moved in this PR and the addition of these unrelated strings is a side effect 👍